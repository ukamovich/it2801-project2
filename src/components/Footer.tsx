function Footer() {
    return (
        <footer className="footer">
            <h2>By Team43</h2>
        </footer>
    );

}
export default Footer