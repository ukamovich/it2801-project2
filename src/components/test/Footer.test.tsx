import Footer from '../Footer';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

//Snapshot test
describe("Header", () => {
  it("renders correctly", () => {
      const tree = renderer.create(
          <Footer/>
          ).toJSON();
          expect(tree).toMatchSnapshot();
  });
});

test('test that it renders with success', () => {
    render(<Footer/>);
  });

  test('test if the name of the footer is in app', () => {
    const { getByText } = render(<Footer />);
    const linkElement = getByText(/By Team43/i); // Must fix this in the end.
    expect(linkElement).toBeInTheDocument();
  });