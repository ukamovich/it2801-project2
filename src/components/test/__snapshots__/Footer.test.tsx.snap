// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<footer
  className="footer"
>
  <h2>
    By Team43
  </h2>
</footer>
`;
